From: Jiri Pirko <jiri@mellanox.com>
Date: Thu, 8 Oct 2015 15:26:17 +0300
Subject: [PATCH 2/7] net/mlx5: Fix typo in mlx5_query_port_pvlc
Patch-mainline: v4.3-rc7
Git-commit: 13b7938883ea6e70e7e23f30dece6f79d7a2961d
References: bnc#923036 FATE#318772

We used the wrong register name for querying the PVLC register

Fixes: a124d13ef59e ('net/mlx5_core: Add more query port helpers')
Signed-off-by: Jiri Pirko <jiri@mellanox.com>
Signed-off-by: Or Gerlitz <ogerlitz@mellanox.com>
Signed-off-by: David S. Miller <davem@davemloft.net>
Acked-by: Cho, Yu-Chen <acho@suse.com>
---
 drivers/net/ethernet/mellanox/mlx5/core/port.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)
--- a/drivers/net/ethernet/mellanox/mlx5/core/port.c
+++ b/drivers/net/ethernet/mellanox/mlx5/core/port.c
@@ -305,7 +305,7 @@ static int mlx5_query_port_pvlc(struct m
 	int err;
 
 	memset(in, 0, sizeof(in));
-	MLX5_SET(ptys_reg, in, local_port, local_port);
+	MLX5_SET(pvlc_reg, in, local_port, local_port);
 
 	err = mlx5_core_access_reg(dev, in, sizeof(in), pvlc,
 				   pvlc_size, MLX5_REG_PVLC, 0, 0);
